src/main/java/org/jenkinsci/plugins/workinghours/WorkingHoursQueueTaskDispatcher.java

```diff
@@ -80,9 +81,13 @@ public CauseOfBlockage canRun(Queue.Item item) {
             Run workflowRun = ((ExecutorStepExecution.PlaceholderTask)item.task).run();
             EnforceScheduleJobProperty prop = workflowJob.getProperty(EnforceScheduleJobProperty.class);
             if (prop != null) {
-                if (!canRunNow(workflowRun, item)) {
-                    log(Level.INFO, "Blocking item %d", item.getId());
-                    return CauseOfBlockage.fromMessage(Messages._WorkingHoursQueueTaskDispatcher_Offline());
+                if (prop.getBranches() == null || prop.getBranches().size() == 0
```

Comment:
Thanks for the PR! 

I'd be tempted to remove `|| prop.getBranches().size() == 0` since it's covered on line 86. It's possibly less efficient (depending on how smart the optimizer is), but not enough to matter,  and having one less condition makes the code a little easier to read IMO. I don't feel super strong about it, so feel free to ignore
```suggestion
                if (prop.getBranches() == null
```